util/make_new_dif/dif_template.h.tpl

```diff
@@ -0,0 +1,209 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+<%doc>
+    This templates requires the following Python objects to be passed:
+
+    1. ip: See util/make_new_dif.py for the definition of the `ip` obj.
+</%doc>
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_${ip.name_upper}_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_${ip.name_upper}_H_
+
+// This file is the "DIF Library header template", which provides a base for
+// building a DIF for a new peripheral, defining all of the declarations that
+// would be expected of a DIF library as described in the README.md.
+//
+// This file should be instantiated with the `util/make_new_dif.py` script.
+//
+// The script also includes additional options for controlling how the template
+// is instantiated. After the script runs, delete this comment, the #error
+// below, and any unnecessary definitions. Remember to run clang-format!
+//
+// Note that this file does not parse as C.
```

Comment:
Does this want to be a mako comment now...?

Reply:
Fixed.